openedx/core/lib/block_structure/tests/helpers.py

```diff
@@ -243,3 +246,29 @@ def assert_block_structure(self, block_structure, children_map, missing_blocks=N
                     set(block_structure.get_parents(block_key)),
                     set(parents),
                 )
+
+
+class EnableTransformerRegistryMixin(object):
+    """
+    Mixin that enables the TransformerRegistry to USE_PLUGIN_MANAGER for
+    finding registered transformers.  USE_PLUGIN_MANAGER is set to False
+    for LMS unit tests to speed up performance of the unit tests, so all
+    registered transformers in the platform do not need to be collected.
+    This Mixin is expected to be used by Tests for integration testing
+    with all registered transformers.
+    """
+    def setUp(self, **kwargs):
+        super(EnableTransformerRegistryMixin, self).setUp(**kwargs)
+        TransformerRegistry.USE_PLUGIN_MANAGER = True
+
+    def tearDown(self):
+        super(EnableTransformerRegistryMixin, self).tearDown()
+        TransformerRegistry.USE_PLUGIN_MANAGER = False
+
+
+def is_course_in_block_structure_cache(course_key, store):
```

Comment:
Can you move this function to the openedx/core/djangoapps directory as well?  The store parameter it takes in is dependent on django.  Although duck-typing potentially allows any non-django-dependent stores to be passed in, this function calls the `make_course_usage_key` method on the store, which isn't defined in the `MockModulestore` defined in this file.  Also, this layer doesn't currently know about courses at all and this function takes in a course_key.